packages/plugins/Solana/src/apis/getFungibleAssets.ts

```diff
@@ -45,19 +45,22 @@ async function getSplTokenList(chainId: ChainId, account: string) {
     const tokenListProvider = new TokenListProvider()
     const provider = await tokenListProvider.resolve()
     const tokenList = provider.filterByChainId(chainId).getList()
-    return data.result
-        .filter((x) => x.account.data.parsed.info.tokenAmount.decimals !== 0) // Filter out non fun.
-        .map((x) => {
-            const info = x.account.data.parsed.info
-            const token = tokenList.find((y) => y.address === info.mint) ?? ({} as TokenInfo)
-            const name = token.name || 'Unknown Token'
-            const symbol = token.symbol || 'Unknown Token'
-            return createFungibleAsset(
-                createFungibleToken(chainId, info.mint, name, symbol, info.tokenAmount.decimals),
-                info.tokenAmount.amount,
-                token.logoURI,
-            )
-        })
+    const splTokens: Web3Plugin.Asset[] = []
+    data.result.forEach((x) => {
+        const info = x.account.data.parsed.info
+        const token = tokenList.find((y) => y.address === info.mint)
```

Comment:
Does it need a Solana flavored `isSameAddress`?

Reply:
Good point.
After doing little research, come out with: address in Solana, or public key, is usually encoded in base64, that means it's case-sensitive.